README.md

```diff
@@ -1,3 +1,61 @@
-# Anoninja
+# Anoninja :octocat:
 
-By Chris and Roselyn
\ No newline at end of file
+Inspired by [Telegraph](https://telegra.ph/), this app allows <del>users</del> ninjas to write down their thoughts whilst remaining anonymous.
+
+Posts are stored in a MongoDB database and can be accessed through an API. This app was created by [@cwesterduin](https://github.com/cwesterduin) and [@roselynle](https://github.com/roselynle) as part of Futureproof's Coding Challenge.
+
+### Installation
+
+-   Clone or download this repo 
+
+### Usage
+
+-   Open your terminal/CLI and navigate to the `anoninja` folder
+-   Ensure docker desktop app is running
+-   Run `docker-compose up` to start the API and DB services
+    -   Access: client on localhost:8080/ and server on localhost:3000/     
+-   Run `docker-compose down` to stop the services and keep data
+-   Run `docker-compose --volumes --remove-orphans` to stop the services and remove all artifacts
```

Comment:
Not a massive thing but would of been nice to use the scripts similar to those found in the assessment for ease of access